lib/elixir_search_extractor_web/plugs/set_current_api_user.ex

```diff
@@ -0,0 +1,15 @@
+defmodule ElixirSearchExtractorWeb.Plugs.SetCurrentApiUser do
```

Comment:
This plus has several naming issues:
1. It should have suffixed with `Plug`
2. It should be nested in a module called `Plugs`
Just like controllers e.g. `ElixirSearchExtractorWeb.KeywordController` are not named `ElixirSearchExtractorWeb.Controllers.Keyword`, this plus should be named `ElixirSearchExtractorWeb.SetCurrentApiUserPlug` 💡 

Same goes

Reply:
Fixed reviews in  [557874f](https://github.com/rafayet-monon/elixir_search_extractor/pull/53/commits/557874f2979622a646f38f0ed77eca6ab39bd25f)